sdk/core/Azure.Core.TestFramework/src/Azure.Core.TestFramework.csproj

```diff
@@ -7,6 +7,8 @@
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
     <PackageReference Include="Azure.Identity" />
+    <PackageReference Include="Azure.AI.Inference" />
```

Comment:
Please revert.  These packages should not be dependencies of the test framework.